docs/en/sql-reference/sql-statements/generated_columns.md

```diff
@@ -10,6 +10,10 @@ You can define one or more generated columns to store the results of expressions
 
 It is also recommended to **evaluate the impact of generated columns on loading performance because computing expressions takes some time**. Additionally, it is advised to **[create generated columns at table creation](#create-generated-columns-at-table-creation-recommended) rather than adding or modifying them after table creation**. Because it is time-consuming and costly to add or modify generated columns after table creation.
 
+However, note that when data loads into tables with a generated column, the time and overhead may increase because StarRocks needs to perform additional computation to calculate the results based on the expression and writes the results into the generated column.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3ad0da73-8c3f-4b21-9976-ae9df9540915)
-> when data is loaded into tables ...